src/components/LoginForm.js

```diff
@@ -4,11 +4,33 @@ import React from 'react';
 //It uses action attribute to POST login
 
 class LoginForm extends React.Component{
+  constructor() {
+    super();
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    fetch('http://localhost:8080/api/login', {
+     method: 'post',
+     headers: {'Content-Type':'application/json'},
+     body: JSON.stringify({
+       "username": this.username.value,
+       "password": this.password.value
+     })
+   }).then((res) => {
+     console.log('success', res.status);
+     alert("super");
+   }).catch((err) => {
+     console.error(err, "asdasd");
+   })
+  }
+
   render() {
     return(
-      <form action="http://localhost:8080/api/login"  method="post" className="loginForm">
```

Comment:
i guess action attribute should be removed from here, handle submit already take care about whole POST request